public/data/games.json

```diff
@@ -0,0 +1,272 @@
+        "image": "images/for_the_king.webp",
+        "links": {
+          "youtube": "https://www.youtube.com/watch?v=w1YNLpTSro0&list=PL1wyuEEMvgudRmssosxsZvoZHNZ3m41bO&pp=gAQBiAQB",
+          "steam": "https://store.steampowered.com/app/527230/For_The_King/"
+        }
+      },
+      {
+        "title": "Lonely Mountains: Downhill",
+        "image": "images/lonely_mountains_downhill.webp",
+        "links": {
+          "youtube": "https://www.youtube.com/watch?v=n21fjVDhneI&list=PL1wyuEEMvguc9-d-9WJXPh5019T91FGNX&pp=gAQBiAQB",
+          "steam": "https://store.steampowered.com/app/711540/Lonely_Mountains_Downhill/"
+        }
+      },
+      {
+        "title": "Red Dead Redemption 2",
+        "image": "images/red_dead_redemption_2.webp",
+        "links": {
+          "youtube": "https://www.youtube.com/watch?v=xOyC8LcIojA&list=PL1wyuEEMvgueiOPVIfKvN41PCJH66eAwQ&pp=gAQBiAQB",
+          "steam": "https://store.steampowered.com/app/1174180/Red_Dead_Redemption_2/"
+        }
+      },
+      {
+        "title": "Slime Rancher",
+        "image": "images/slime_rancher.webp",
+        "links": {
+          "youtube": "https://www.youtube.com/watch?v=PLN7v5KtfU8&list=PL1wyuEEMvguftV26HpQMxAuLvfKxdM4_u&pp=gAQBiAQB",
+          "steam": "https://store.steampowered.com/app/433340/Slime_Rancher/"
+        }
+      },
+      {
+        "title": "Portal",
+        "image": "images/portal.webp",
+        "links": {
+          "youtube": "https://www.youtube.com/watch?v=OJLEsl7YLbs&list=PL1wyuEEMvgudp4e0VR2WciQuZZMxGLrvo&pp=gAQBiAQB",
+          "steam": "https://store.steampowered.com/app/400/Portal/"
+        }
+      },
+      {
+        "title": "Beyond: Two Souls",
+        "image": "images/beyond_two_souls.webp",
+        "links": {
+          "youtube": "https://www.youtube.com/watch?v=fH8ojr-7N1o&list=PL1wyuEEMvgudx7AefjWRZxn5qns8TZNaY&pp=gAQBiAQB",
+          "steam": "https://store.steampowered.com/app/960990/Beyond_Two_Souls/"
+        }
+      },
+      {
+        "title": "FOR HONOR™",
+        "image": "images/for_honor.webp",
+        "links": {
+          "youtube": "https://www.youtube.com/watch?v=0aBLPX1yLqw&list=PL1wyuEEMvgueIj18C1IsITkl3N1hVhIIZ",
+          "steam": "https://store.steampowered.com/app/304390/FOR_HONOR/"
+        }
+      },
+      {
+        "title": "Mutant Year Zero: Road to Eden",
+        "image": "images/mutant_year_zero_road_to_eden.webp",
+        "links": {
+          "youtube": "https://www.youtube.com/watch?v=kDgakvGjNdA&list=PL1wyuEEMvgucLnOuk1Sz2X0COf9aHUNhw&pp=gAQBiAQB",
+          "steam": "https://store.steampowered.com/app/760060/Mutant_Year_Zero_Road_to_Eden/"
+        }
+      },
+      {
+        "title": "",
+        "image": "images/.webp",
+        "links": {
+          "youtube": "",
+          "steam": ""
+        }
+      },
+      {
+        "title": "",
+        "image": "images/.webp",
+        "links": {
+          "youtube": "",
+          "steam": ""
+        }
```

Comment:
@Florian-Dj  Just to let you know when there is no title on the data, the card won't appear.